v2020.0/Actuator.py

```diff
@@ -59,8 +59,10 @@
 	from gpiozero import TimeOfDay
 
 	# Allow control of output devices such as Motors, Servos, LEDs, and Relays
-	from gpiozero import Motor, Servo, LED, Energenie, OutputDevice
-
+	from gpiozero import Motor, Servo, LED, Energenie, OutputDevice, AngularServo
+	import gpiozero
+	from gpiozero.pins.mock import MockFactory
+	gpiozero.Device.pin_factory = MockFactory()
```

Comment:
What does this allow?

Reply:
There’s also a gpiozero.pins.mock.MockFactory which generates entirely fake pins. This was originally intended for GPIO Zero developers who wish to write tests for devices without having to have the physical device wired into their Pi. 
source :
https://gpiozero.readthedocs.io/en/stable/api_pins.html